andela/travel_tool_back
# #160371193 Edit Travel Request Notifications

#### What does this PR do?
Create endpoints for notifications

#### Description of Task to be completed?
-  create a model for notifications
- create POST /notification endpoint to store notifications

#### How should this be manually tested?
- Clone the repo -git clone https://github.com/andela/travel_tool_back.git
- Enter into the project directory - cd travel_tool_back
- Start the app using make start
- Login on your browser and grab your token from the application tab in Developers tab
- Open Postman
- Create `POST` a travel request using the field shown on the screenshot below using `localhost:5000/requests`
- Edit `PUT` a travel request and save using `localhost:5000/requests/:requestId`
- Check your `Notifications table` on the database to see the notification created after submitting the edited travel request

#### Any background context you want to provide?
- RequestId is generated after creating a new request. Check the response to get the `requestId`
- Make sure the manager name is on the database before u create a new request
- I only worked on RequestControllers.js file under modules, the other files are from [this](https://github.com/andela/travel_tool_back/pull/44) story

#### What are the relevant pivotal tracker stories?
[#160371193](https://www.pivotaltracker.com/story/show/160371193)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-19 at 9 35 21 pm" src="https://user-images.githubusercontent.com/28366272/45779806-f562cb00-bc53-11e8-8f78-e345e1aa7838.png">


#### Questions:
N/A
